ui/main.qml

```diff
@@ -317,8 +318,20 @@ StatusWindow {
 
     Component {
         id: appLoadingAnimation
-        SplashScreen {
```

Comment:
I guess we can remove the SplashScreen component?

Reply:
Well.. maybe yes, but then also the `status_splash_*.gif`?

Reply:
Hmm, or even better, I'll move this implementation from main.qml into SplashScreen

Reply:
Done